### PR TITLE
[Threading][test] Repair none threading tests

### PR DESCRIPTION
**Pull request by kateinoigakukun**
This fixes the all tests on no-thread based on https://github.com/apple/swift/pull/59403

### CODE REVIEW

**Comment by kateinoigakukun** [comment]
@swift-ci Please smoke test

**Comment by kateinoigakukun** [comment]
preset=buildbot_incremental_linux_crosscompile_wasm
@swift-ci Please test with preset Linux Platform

**Comment by kateinoigakukun** [comment]
preset=buildbot_incremental_linux_crosscompile_wasm
@swift-ci Please test with preset Linux Platform

**Comment by kateinoigakukun** [comment]
@swift-ci Please smoke test

**Review by al45tair — COMMENTED** [review]
Mostly looks fine, though I'm not sure we should remove the `threading_package_default` line in `test/CMakeLists.txt`. We should be passing the threading package name that we expect to use through to lit.

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -398,7 +398,6 @@ foreach(SDK ${SWIFT_SDKS})
         if("${SWIFT_SDK_${SDK}_THREADING_PACKAGE}")
           list(APPEND LIT_ARGS "--param" "threading=${SWIFT_SDK_${SDK}_THREADING_PACKAGE}")
         else()
-          threading_package_default("${SDK}" SWIFT_THREADING_PACKAGE)
```
[the remark this answers is not in the document]
Comment (kateinoigakukun) [reply test/CMakeLists.txt:401]: Oh! Thank you for your quick fix 👍

**Comment by MaxDesiatov** [comment]
@swift-ci please smoke test

**Comment by kateinoigakukun** [comment]
preset=buildbot_incremental_linux_crosscompile_wasm
@swift-ci Please test with preset Linux Platform

**Comment by kateinoigakukun** [comment]
@swift-ci Please smoke test

**Review by al45tair — APPROVED** [review]

**Comment by kateinoigakukun** [comment]
Thank you for quick review 😄

**Comment by al45tair** [comment]
😊 Thank-you for working on this — and I'm glad the threading package stuff is useful for you.

**Pull request merged**